pages/admin.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement, useState } from 'react'
+import { useTheme } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
+import { LogLevel } from '@/types/interfaces'
+import Wrapper from '../src/Wrapper'
+import FilterMenu, { LogFilters } from '../src/FilterMenu/FilterMenu'
+import LogTree from '../src/LogTree/LogTree'
+
+export default function Admin(): ReactElement {
+  const theme = useTheme()
+  const [logFilters, setLogFilters] = useState<LogFilters>({
+    level: LogLevel.TRACE,
+    buildId: '',
+    requestId: '',
+    search: '',
+    isRegex: false,
+  })
+
+  const handleGetLogs = (filters: LogFilters): void => {
+    setLogFilters(filters)
+  }
+
+  return (
+    <Wrapper title='Admin' page='admin'>
```

Comment:
We should check the user role here as well and return an error message. Right now if you view the page as a non-admin you just get the log box with a "loading..." text.

Reply:
Good catch, I'll put some generic unauthorised message in the event that someone manually navigates to the page.